### ISSUE TITLE
[feature request] CTW effect when dialog launches

### ISSUE THREAD

**Issue by benoitheylens** [Lanrutcon/CatchTheWind#19]
I had an idea (bad news for you :p).

Is is possible, as it works for quest, that CTW displays the dialog that opens when you click and the bubbles ?

Would be awesome (again).

![wowscrnshot_091116_082321](https://cloud.githubusercontent.com/assets/21984111/18415781/c3b9c5b8-77fe-11e6-8f22-ee0f93ccd7dd.jpg)

**Comment by chevex** [comment]
I personally miss when CTW would activate for every NPC and not just quest ones, but I know some people hated it. Would be cool to have that be an option I could re-enable.

**Comment by benoitheylens** [comment]
I don't necessary want it for every npc, and maybe it's not feasible but sometimes, these kind of big chunk of text are in a whole block. Maybe it's not easy/feasible to split this block in shorter blocks or sentences.

**Comment by Lanrutcon** [comment]
Well, currently the addon is completely focused to quest interaction.
In the first weeks of the addon, someone suggested to create the whole NPC interaction, just like [Storyline](https://mods.curse.com/addons/wow/total-rp-3-storyline) does.

The main goal is to make quest interaction as smooth and elegant as possible, always providing all the information that the player needs. Currently, there are still some stuff missing (e.g. Quest Title - still thinking about where to put it...), after that I'm sure that I'll expand the addon.


`I don't necessary want it for every npc, and maybe it's not feasible but sometimes, these kind of big chunk of text are in a whole block. Maybe it's not easy/feasible to split this block in shorter blocks or sentences.`

The addon separates the text in paragraphs. From what I've seen (and until now, no player reported), there was no paragraph that was too big that would go out of screen.
If this problem arises, the first solution would be decreasing the font size, but since I didn't get any complain about it, I won't bother too much until I see it.


So, this feature will be implemented but there still a "long way to walk" before it.

**Comment by chevex** [comment]
Yeah I understand, I just liked it when [dialog sequences like these](https://youtu.be/ce2NudWUfvo) were also handled by CTW. Much more immersive, but I understand if it's an issue for others.
